### ISSUE TITLE
Questions about the training

### ISSUE THREAD

**Issue by 2212582443** [JoshuaChou2018/SkinGPT-4#11]
Did you just fine-tune the Alignment layer but freeze both the Image Embedding and LLM like what MiniGPT-4 did?
How did you convert the Phase 1 dataset into VQA？The SKINCON dataset only provides pictures and the coding of  medical concepts contained in the corresponding picture.

**Comment by JoshuaChou2018** [comment]
Yes, for this released version of codes, you just need to finetune the alignment layer. For the SKINCON dataset in phase 1, you need to convert it into:
V: Image
Q: Could you please describe the skin disease image? / What is wrong with my skin?
A: This is a skin disease image with the following medical features (clinical concepts): Fissure, Induration, Xerosis, Telangiectasia, Scale, Scar, Friable, Sclerosis, ....

For phase 2 finetuning, you will need to have some well-annotated data with medical features and differential diagnosis in this format:
V: Image
Q: Could you please describe the skin disease image? / What is wrong with my skin?
A: This is a skin disease image with the following medical features (clinical concepts): Fissure, Induration, Xerosis, Telangiectasia, Scale, Scar, Friable, Sclerosis, ..... This could be a skin disease image of .... (For the future version we are working currently, we add additional reasoning annotation)

**Comment by 2212582443** [comment]
Thank you, I completely understand what you mean. Looking forward to your future version.

**Comment by 2212582443** [comment]
Can you provide your script for processing public data sets during training?
